Login/Registration Update

dblixt opened the pull request:
Created Login/Registration page.
Created registerDOA.
Reworked RegisterController and User classes.

dblixt committed:
created login/registration page

dblixt committed:
updated loginpage

dblixt committed:
Updated Registration/ Login Page

Updated registration/login page with html and css. Created
RegistrationDOA and User. Reworked RegistrationController.

dblixt committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into db9

danielnilsson9 commented:
registerDAO bör heta som de andra DAO objecten, den bör också endast arbeta med en tabell på samma sätt som dom andra.

Den var tänkt att arbeta med Users tabeller, så namn borde vara UsersDAO.

Den bör ej heller göra några insättningar i Profiles tabeller, det är tänkt att PorfilesDAO ska hantera dessa.

dblixt committed:
DOA Updates

Renamed registerDOA to UserDOA.
Made some changes in UserDOA and ProfileDOA.
Reworked RegisterController to fit changes in DOA classes.
